### PR TITLE
Editable gearsets

### PR DESCRIPTION
**Pull request by Juanaxu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ run `python main.py` then `run_test()`
 
 * aard appel (famfrit)
 * HI MAYBE YOUR NAME HERE? potato 3059 on discord
+* Rhiow Silrah (Gilgamesh)
```
Comment (ackao) [inline README.md:19]: swap order of last 2 bullet points

File: client/src/app/data/gearset.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	mainStat: number;
+	dh: number;
+	crit: number;
+	det: number;
+	sps: number;
+	pie: number;
+	gcd: number;
+	mp: number;
+	dps: number;
+}
```
Comment (ackao) [inline client/src/app/data/gearset.ts:14]: nit: newline

File: client/src/app/gearset-list/gearset-list.component.ts [hunk 3]
```diff
@@ -59,10 +50,39 @@ export class GearsetListComponent implements OnInit {
 		mp: -1491.90,
 		dps: 13621.75,
 	}];
+	editCache: { [key: string]: { edit: boolean; data: GearSet } } = {};
```
Comment (ackao) [inline client/src/app/gearset-list/gearset-list.component.ts:53]: this is good for now but we'll want to use a backend database soon

**Review by ackao — CHANGES_REQUESTED** [review]

File: client/src/app/gearset-list/gearset-list.component.ts (hunk 3 above)
Comment (Juanaxu) [reply client/src/app/gearset-list/gearset-list.component.ts:53]: Yeah, saving the gearsets permanently would be A+

File: README.md (hunk 1 above)
Comment (Juanaxu) [reply README.md:19]: Done

File: client/src/app/data/gearset.ts (hunk 2 above)
Comment (Juanaxu) [reply client/src/app/data/gearset.ts:14]: Done

**Review by ackao — APPROVED** [review]

**Pull request merged**